.env.example

```
@@ -17,3 +17,8 @@ MAIL_HOST=localhost
 MAIL_PORT=1025
 MAILGUN_API_KEY=
 MAILGUN_DOMAIN=
+
+# S3 Media Storage
+S3_BUCKET_NAME=mpgagolf
+S3_REGION=us-west-2
+S3_MEDIA_PREFIX=media
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_MEDIA_PREFIX key should go before the S3_REGION key

(UnorderedKey)